### DISCUSSION TITLE
redirect=click2load.html

### DISCUSSION THREAD

**Discussion by dportvine** [uBlockOrigin/uAssets#19732] [Q&A]
I have three sites where developers have added a lot of youtube frames. This leads to a large load on the CPU. I used this rule, but I did not get the result I expected. Can you help me? Chrome only.

```adb
||youtube.com/embed/$subdocument,third-party,redirect=click2load.html,domain=andro-news.com|novadoba.com.ua|ukrinform.ua
```

`https://andro-news.com/ua/news/poco-m6-pro-legend-vibe.html`

<details>
<summary>Screen</summary>

![2023-09-14_075343](https://github.com/uBlockOrigin/uAssets/assets/111344219/25bd7977-e5ac-4617-9993-03f610fdb100)
</details>

`https://novadoba.com.ua/`

<details>
<summary>Screen</summary>

![2023-09-14_075433](https://github.com/uBlockOrigin/uAssets/assets/111344219/935f05c1-4967-4577-aaf8-4ae8382bdadc)
</details>

`https://www.ukrinform.ua/`

<details>
<summary>Screen</summary>

![2023-09-14_075511](https://github.com/uBlockOrigin/uAssets/assets/111344219/ba75dbff-31d4-4f24-89c1-438e84c8af9f)
</details>

**Comment by ghost** [comment]
https://github.com/uBlockOrigin/uBlock-issues/issues/1938

***

Even the solution works:

    andro-news.com,novadoba.com.ua,ukrinform.ua##+js(aopr,  window.chorus_lazy_assets)

**Reply by dportvine** [reply]
It doesn't works for me.

<details>
<summary>Screen</summary>

![2023-09-14_103156](https://github.com/uBlockOrigin/uAssets/assets/111344219/eb98a16a-64bf-4108-a7d7-ecb28f630c20)
</details>

**Reply by stephenhawk8054** [reply]
Blocking

```adb
||andro-news.com/skins/news2/all.js$script
```

works for me, but I'm lazy to find which causes issue in that script

**Reply by dportvine** [reply]
Yep it works for me too.
